binder/binderlocationbackend.cpp

```diff
@@ -221,6 +221,8 @@ void processNmea(gint64 timestamp, const char *nmeaData)
         parseRmc(nmea);
 }
 
+const double KnotsToMps = 0.514444;
```

Comment:
This should probably be static.

Reply:
Same as above, copied from hal backend. Maybe also in this should be changed in both backends.